e2e_tests/integrations/test_huggingface.py

```diff
@@ -167,43 +171,69 @@ def test_run_access_without_callback_configured(self):
         with pytest.raises(Exception):
             NeptuneCallback.get_run(trainer)
 
-    def test_log_parameters_indicator(self, environment):
+    def _test_with_run_initialization(self, environment, pre, post):
```

Comment:
Can we do `def _test_with_run_initialization(self, environment, *, pre, post):` to force keyword arguments for callbacks, it'll be easier to read following code?

Reply:
Done